sdks/java/core/src/main/java/org/apache/beam/sdk/expansion/ExternalTransformRegistrar.java

```diff
@@ -30,5 +30,5 @@
 public interface ExternalTransformRegistrar {
 
   /** A mapping from URN to an {@link ExternalTransformBuilder} class. */
-  Map<String, Class<? extends ExternalTransformBuilder>> knownBuilders();
+  Map<String, Class<? extends ExternalTransformBuilder<?, ?, ?>>> knownBuilders();
```

Comment:
FYI @sclukas77 you may run into some merge conflicts with this PR when you change this interface to `Map<String, ExternalTransformBuilder>`